### PR TITLE
GROW-395 MG Homepage Sticky Selector

### PR DESCRIPTION
**Pull request by eddieferrer**
* Implemented MG Homepage sticky selector component
* Minor fixes around MG homepage

STILL TODO:
* Add icons to cause buttons
* Sort cause buttons according to order in design
* Finalize tracking params
* Fix sticky positioning on window resize

### CODE REVIEW

File: src/pages/Homepage/MonthlyGoodHomepage.vue [hunk 1]
```diff
@@ -5,12 +5,21 @@
 				<div
 					class="small-12 medium-10 large-6 xlarge-5 small-order-2 large-order-1 columns"
 				>
-					<img
-						v-if="heroImage.url"
-						class="featured-loans__img"
-						:src="heroImage.url"
-						:alt="heroImage.description"
+					<router-link
+						:to="`/lend-by-category`"
```
Comment (eddieferrer) [inline src/pages/Homepage/MonthlyGoodHomepage.vue:9]: loan card should link to 'lend-by-category' - final tracking params on this page tbd

File: src/components/Homepage/MonthlyGoodSelector.vue [hunk 2]
```diff
@@ -0,0 +1,284 @@
+<template>
+	<div class="monthly-selector">
+		<div class="row align-spaced collapse">
+			<div class="column">
+				<button
+					class="monthly-selector__button"
+					:class="causeClass"
+					@click="openClause"
```
Comment (mcstover) [inline src/components/Homepage/MonthlyGoodSelector.vue:8]: "cause" or "clause"

**Review by mcstover — APPROVED** [review]
Overall great work moving this forward. I'm on the fence with using button list instead an actual form with radio groups but maybe that's something we can revisit later if this ends up be used elsewhere.

File: src/components/Homepage/MonthlyGoodSelector.vue (hunk 2 above)
Comment (eddieferrer) [reply src/components/Homepage/MonthlyGoodSelector.vue:8]: > "cause" or "clause"

😆

**Pull request merged**